### PR TITLE
Porting isolated changes

### PR DESCRIPTION
**Pull request by mike-lvov**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1548

### CODE REVIEW

File: scripts/upload-table-config.ts [hunk 1]
```diff
@@ -19,8 +19,8 @@ import {
 // ---------------------------------------------------------
 
 const newTables: Table[] = [
-  ...generateTables({ num: 5, capacity: 6 }),
-  ...generateTables({ num: 5, capacity: 2, startFrom: 5, columns: 2 }),
+  ...generateTables({ num: 5 }),
+  ...generateTables({ num: 5, startFrom: 5, columns: 2 }),
```
Comment (colinhowe) [inline scripts/upload-table-config.ts:23]: Was this change intentional?

Comment (mike-lvov) [reply scripts/upload-table-config.ts:23]: Yes, the capacity is now calculated based on columns\rows number 
image

**Review by colinhowe — APPROVED** [review]
LGTM
